kgforge/specializations/models/demo_model.py

```diff
@@ -81,7 +81,7 @@ def mapping(self, entity: str, source: str, type: Callable) -> Mapping:
 
     # Validation.
 
-    def _validate_one(self, resource: Resource) -> None:
+    def _validate_one(self, resource: Resource, type_: str=None) -> None:
         # If the resource is not typed, AttributeError is raised: run() sets _validated to False.
```

Comment:
This text should be adapted to mention 'type_'

Reply:
I pushed a change that changes the text